nixos/modules/services/networking/firefox/sync-server.nix

```diff
@@ -124,7 +124,13 @@ in
       description = "Firefox Sync Server";
       wantedBy = [ "multi-user.target" ];
       path = [ pkgs.pythonPackages.pasteScript pkgs.coreutils ];
-      environment.PYTHONPATH = "${pkgs.pythonPackages.syncserver}/lib/${pkgs.pythonPackages.python.libPrefix}/site-packages";
+      environment.PYTHONPATH = let
+          penv = pkgs.python.buildEnv.override {
+            extraLibs = [ pkgs.pythonPackages.syncserver ];
+            ignoreCollisions = true;
+          };
+        in "${penv}/${pkgs.python.sitePackages}/";
```

Comment:
Indeed, `paster` and `syncserver` are two separate Python programs. They should not share any `PYTHONPATH` at all. Therefore, exporting `PYTHONPATH` is tricky. 

Could you try https://github.com/FRidh/nixpkgs/commit/9687f593f81b71912749f3778dd542a3703e9f90